Content.Server/Fax/FaxSystem.cs

```diff
@@ -397,12 +398,17 @@ public void Send(EntityUid uid, FaxMachineComponent? component = null, EntityUid
             { FaxConstants.FaxPaperContentData, paper.Content },
         };
 
+        if (metadata.EntityPrototype != null)
+        {
+            payload[FaxConstants.FaxPaperPrototypeData] = metadata.EntityPrototype.ID;
+        }
+
```

Comment:
This should just copy visuals instead; if an entity is constructed entirely ingame / in code vs prototype it should function the same and not rely on hardcoding prototype ID copies.

Reply:
Alright having talked to Jez archetypes has entity copies which is what it should do instead so this just needs a TODO to not do this ID check anymore.